namespaces/mson-namespace.md

```diff
@@ -54,6 +54,78 @@ In other words, an expanded element is one that does not contain any _Identifier
 
 The expanded Refract MUST, however, keep the track of what data structure was expanded and what from where.
 
+### Example
+
+Extending the element "A" to form new element "B":
+
+```json
+{
+  "element": "extend",
+  "meta": {
+    "id": "B"
+  },
+  "content": [
+    {
+      "element": "string",
+      "meta": {
+        "id": "A"
+      },
+      "content": "base element content"
+    },
+    {
+      "element": "string",
+      "content": "derived content"
+    }
+  ]
+}
+```
+
+In MSON refract, the following is equivalent:
```

Comment:
I think it would be helpful to be more descriptive here. Why is this equivalent? It's because in MSON, inheritance is implicitly defined.